src/openms/source/TRANSFORMATIONS/FEATUREFINDER/EGHTraceFitter.cpp

```diff
@@ -187,6 +185,7 @@ namespace OpenMS
     this->apex_rt_ = source.apex_rt_;
```

Comment:
did you check if the whole function is just the default copy assignment and we could write = default in the header? same for copy constructor

Reply:
it has update_members